Modules/Module_Processing/Module_Population/xASL_stat_GetDICOMStatistics.m

```diff
@@ -92,9 +92,13 @@ function xASL_stat_GetDICOMStatistics(x, ScanType, HasSessions, bOverwrite)
 end
 fprintf('\n');
 
-
+% Verify that all gathered cells include strings and not cells inside
+listCells = find(cellfun(@iscell, TSV(:)));
```

Comment:
I thought I fixed this with the xASL_num2str or something. But if this fix by you works and you tested it, I'm fine.